control-plane/pkg/reconciler/consumergroup/consumergroup.go

```diff
@@ -146,10 +150,12 @@ func (r Reconciler) ReconcileKind(ctx context.Context, cg *kafkainternals.Consum
 	}
 	cg.MarkReconcileConsumersSucceeded()
 
+	r.maybeRequeueOnPendingReplicas(cg)
+
 	return nil
 }
 
-func (r Reconciler) FinalizeKind(ctx context.Context, cg *kafkainternals.ConsumerGroup) reconciler.Event {
+func (r *Reconciler) FinalizeKind(ctx context.Context, cg *kafkainternals.ConsumerGroup) reconciler.Event {
```

Comment:
just curious, why was `ConsumerGroup` not using points before, like the other reconcilers, on the code base, including `consumer`?